src/Tools/dotnet-monitor/Program.cs

```diff
@@ -39,7 +39,7 @@ class Program
                 aliases: new[] { "-u", "--urls" },
                 description: "Bindings for the REST api.")
             {
-                Argument = new Argument<string[]>(name: "urls", getDefaultValue: () => new[] { "http://localhost:52323" })
+                Argument = new Argument<string[]>(name: "urls", getDefaultValue: () => new[] { "https://localhost:52323" })
```

Comment:
I assume that in order to allow dotnet-monitor to run without https, then the urls parameter must be specified e.g. `--urls http://localhost:52323`. Could you consider adding an environment variable to the configuration that allows specification of the URLs so that the command line doesn't need to be recreated for Docker containers in AKS?